install/running_install.adoc

```diff
@@ -403,6 +403,32 @@ ifdef::openshift-origin[]
 endif::[]
 ----
 
+[running-the-advanced-installation-openstack]
+==== Running the Installation Playbook for OpenStack
+
+[IMPORTANT]
+====
+The OpenStack installation playbook is a Technology Preview feature.
+ifdef::openshift-enterprise[]
+Technology Preview features are not supported with Red Hat production service
+level agreements (SLAs), might not be functionally complete, and Red Hat does
+not recommend to use them for production. These features provide early access to
+upcoming product features, enabling customers to test functionality and provide
+feedback during the development process.
+
+For more information on Red Hat Technology Preview features support scope, see
+https://access.redhat.com/support/offerings/techpreview/.
+endif::[]
+====
+
+If you want to install {product-title} on an existing OpenStack installation,
+you can do so by using the playbook for OpenStack. Run the following command:
+
+----
+# rpm -qf /usr/share/ansible/openshift-ansible/playbooks/openstack
+    openshift-ansible-playbooks-3.9.14-1.git.3.c62bc34.el7.noarch
```

Comment:
This command only shows the package that owns the playbook directory.  @tomassedovic, could you provide the exact command we should document?  I'm guessing it may be as simple as just running ansible-playbook and passing it in but I'm curious if there are any other required inventory variables.

Reply:
@jianlinliu  Let Jianlin to check if it's something OpenShift QE has been covering or if we need to re-direct to OpenStack QE.

Reply:
It is better to re-direct this to OpenStack QE.

Reply:
@vikram-redhat, do we have a contact in OpenStack QE?